src/platform/apollolake/apollolake.x.in

```diff
@@ -470,6 +470,13 @@ SECTIONS
     KEEP (*(.fw_ready))
   } >sof_fw :sof_fw_phdr
 
+  .comp_init : ALIGN(4)
```

Comment:
Is it possible to pack all RO (rodata and other) items first and then RW (data)?

Reply:
Sure, its done (If I understood you correctly)